acceptance/openstack/identity/v3/identity.go

```diff
@@ -166,3 +182,71 @@ func DeleteDomain(t *testing.T, client *gophercloud.ServiceClient, domainID stri
 
 	t.Logf("Deleted domain: %s", domainID)
 }
+
+// UnassignRoleFromUserOnProject will revoke a role of a user on a project. A fatal error will
+// occur if the revoke was unsuccessful. This works best when used as a deferred function
+func UnassignRoleFromUserOnProject(t *testing.T, client *gophercloud.ServiceClient, role *roles.Role, user *users.User, project *projects.Project) {
+	t.Logf("Attempting to remove role %s from user %s on project %s", role.Name, user.Name, project.Name)
+
+	err := roles.UnassignFromUserOnProject(client, role.ID, user.ID, project.ID).ExtractErr()
+	if err != nil {
+		t.Fatalf("Unable to remove role")
```

Comment:
nit: maybe return error instead of Fatalf. Also please add role.ID/user.ID to the log message.

Reply:
UnassignFromUserOnProject is most likely to be used as a deferred function, so I created a fatal here.

Reply:
@dongwoo1005 Is similar being done for other *remove*/*cleanup* kind of functions in the codebase? I worry that an assumption is being made about how this function would be invoked by caller. Perhaps some callers just want to get a regular error like all other functions and deal with it in their layer.